app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,26 @@
+<h2>Log in</h2>
+
+<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %><br />
+    <%= f.password_field :password, autocomplete: "current-password" %>
+  </div>
+
+  <% if devise_mapping.rememberable? %>
+    <div class="field">
+      <%= f.check_box :remember_me %>
+      <%= f.label :remember_me %>
+    </div>
+  <% end %>
+
+  <div class="actions">
+    <%= f.submit "Log in" %>
+  </div>
+<% end %>
+
+<%= render "devise/shared/links" %>
```

Comment:
- [OPTIONAL] I want to suggest that the views in devise for the registration and login should be modified to match the wireframes and their styling. Please see the screenshots below.

| Your Design |
:------------:|
![Screen Shot 2022-10-06 at 6 47 22 PM](https://user-images.githubusercontent.com/39523312/194395073-e4c1c975-5f75-449e-a095-2c7ce2880933.png) |

 | Expected Wirefarme |
:------------------------:|
 ![Screen Shot 2022-09-28 at 11 22 26 PM](https://user-images.githubusercontent.com/39523312/194395166-cff07833-f3c5-4b32-a4b2-0d928de9882a.png) |

Reply:
Thanks for mentioning